### PR TITLE
RHCLOUD-31838 add new simple widgets

### PR DESCRIPTION
**Pull request by aferd**
Pull in three new widgets : 
Quay.io
OpenShift AI
Advanced Cluster Security

### CODE REVIEW

File: rest/models/DashboardTemplate.go [hunk 1]
```diff
@@ -50,17 +50,21 @@ const (
 	Edge                AvailableWidgets = "edge"
 	Ansible             AvailableWidgets = "ansible"
 	Rhel                AvailableWidgets = "rhel"
-	Openshift           AvailableWidgets = "openshift"
+	OpenShift           AvailableWidgets = "openshift"
 	RecentlyVisited     AvailableWidgets = "recentlyVisited"
+	OpenShiftAi         AvailableWidgets = "openshiftAi"
+	Quay	            AvailableWidgets = "quay"
+	Acs           		AvailableWidgets = "acs"
```
Comment (aferd) [inline rest/models/DashboardTemplate.go:57]:
```suggestion
	Acs         AvailableWidgets = "acs"
```

Comment (aferd) [inline rest/models/DashboardTemplate.go:57]:
```suggestion
	Acs         AvailableWidgets = "acs"
```

File: rest/models/DashboardTemplate.go [hunk 2]
```diff
@@ -242,15 +246,18 @@ const (
 	OpenShiftIcon        WidgetIcons = "OpenShiftIcon"
 	EdgeIcon             WidgetIcons = "EdgeIcon"
 	AnsibleIcon          WidgetIcons = "AnsibleIcon"
+	QuayIcon			 WidgetIcons = "QuayIcon"
```
Comment (aferd) [inline rest/models/DashboardTemplate.go:249]:
```suggestion
	QuayIcon	  WidgetIcons = "QuayIcon"
```

File: rest/models/DashboardTemplate.go [hunk 3]
```diff
@@ -242,15 +246,18 @@ const (
...
+	ACSIcon				 WidgetIcons = "ACSIcon"
```
Comment (aferd) [inline rest/models/DashboardTemplate.go:250]:
```suggestion
	ACSIcon		WidgetIcons = "ACSIcon"
```

**Review by epwinchell — APPROVED** [review]

File: rest/models/DashboardTemplate.go [hunk 4]
```diff
@@ -242,15 +246,18 @@ const (
...
 
 func (wi WidgetIcons) IsValid() error {
 	switch wi {
-	case BellIcon, HistoryIcon, OutlinedBookmarkIcon, RocketIcon, StarIcon, RhelIcon, OpenShiftIcon, EdgeIcon, AnsibleIcon:
+	case BellIcon, HistoryIcon, OutlinedBookmarkIcon, RocketIcon, StarIcon, RhelIcon, OpenShiftIcon, EdgeIcon, AnsibleIcon, QuayIcon, ACSIcon, OpenShiftAiIcon:
 		return nil
 	}
 
-	return fmt.Errorf("invalid widget icon. Expected one of %s, %s, %s, %s, %s, %s, %s, %s, %s, got %s", BellIcon, HistoryIcon, OutlinedBookmarkIcon, RocketIcon, StarIcon, RhelIcon, OpenShiftIcon, EdgeIcon, AnsibleIcon, wi)
+	return fmt.Errorf("invalid widget icon. Expected one of %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s got %s", BellIcon, HistoryIcon, OutlinedBookmarkIcon, RocketIcon, StarIcon, RhelIcon, OpenShiftIcon, EdgeIcon, AnsibleIcon, QuayIcon, ACSIcon, OpenShiftAiIcon wi)
```
Comment (karelhala) [inline rest/models/DashboardTemplate.go:260]: Missing column
```suggestion
	return fmt.Errorf("invalid widget icon. Expected one of %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s got %s", BellIcon, HistoryIcon, OutlinedBookmarkIcon, RocketIcon, StarIcon, RhelIcon, OpenShiftIcon, EdgeIcon, AnsibleIcon, QuayIcon, ACSIcon, OpenShiftAiIcon, wi)
```

**Review by karelhala — APPROVED** [review]
Looking good, just one missing column.

File: rest/service/dashboardTemplate.go [hunk 5]
```diff
@@ -128,6 +128,33 @@ var (
 				Title: "Recently visited",
 			},
 		},
+		models.Openshift: models.ModuleFederationMetadata{
```
Comment (karelhala) [inline rest/service/dashboardTemplate.go:131]: The build is failing in these models, I guess we need to use capital `S`?
```suggestion
		models.OpenShift: models.ModuleFederationMetadata{
```

File: rest/service/dashboardTemplate.go [hunk 6]
```diff
@@ -128,6 +128,33 @@ var (
...
+			Scope:    "landing",
+			Module:   "./OpenShiftAiWidget",
+			Defaults: models.BaseWidgetDimensions.InitDimensions(models.BaseWidgetDimensions{}, 1, 3, 3, 1),
+			Config: models.WidgetConfiguration{
+				Icon:  models.OpenShiftAiIcon,
+				Title: "Red Hat OpenShift AI",
+			},
+		},
+		models.Openshift: models.ModuleFederationMetadata{
```
Comment (karelhala) [inline rest/service/dashboardTemplate.go:140]: Shouldn't this be `Quay`
```suggestion
		models.Quay: models.ModuleFederationMetadata{
```

File: rest/service/dashboardTemplate.go [hunk 7]
```diff
@@ -128,6 +128,33 @@ var (
...
+			Scope:    "landing",
+			Module:   "./QuayWidget",
+			Defaults: models.BaseWidgetDimensions.InitDimensions(models.BaseWidgetDimensions{}, 1, 3, 3, 1),
+			Config: models.WidgetConfiguration{
+				Icon:  models.QuayIcon,
+				Title: "Quay.io",
+			},
+		},
+		models.Openshift: models.ModuleFederationMetadata{
```
Comment (karelhala) [inline rest/service/dashboardTemplate.go:149]: Shouldn't this be `Acs`?
```suggestion
		models.Acs: models.ModuleFederationMetadata{
```

File: rest/models/DashboardTemplate.go [hunk 8]
```diff
@@ -242,15 +246,18 @@ const (
...
```
Comment (karelhala) [inline rest/models/DashboardTemplate.go:260]:
```suggestion
	return fmt.Errorf("invalid widget icon. Expected one of %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s got %s", BellIcon, HistoryIcon, OutlinedBookmarkIcon, RocketIcon, StarIcon, RhelIcon, OpenShiftIcon, EdgeIcon, AnsibleIcon, QuayIcon, ACSIcon, OpenShiftAiIcon, wi)
```

**Pull request merged**